FreeTubeApp/FreeTube
# Parse IV community playlist attachments

# Parse IV community playlist attachments

## Pull Request Type

## Related issue
Relies on https://github.com/iv-org/documentation/pull/378 (tested with an invidious docker instance)

## Description
This PR will allow playlists to be shown for Invidious community posts

## Testing 
- point to an invidious instance that supports the new playlist parsing
- go to a channel that has playlists (example TBD)
- See it load

## Desktop
- **OS:** Windows
- **OS Version:** 11
- **FreeTube version:** 0.18.0
